### PR TITLE
Document log level format

### PR DESCRIPTION
**Pull request by raxod502-plaid**
Documentation mentioned that log level could be customized via an environment variable, but didn't explain what values could be assigned to that environment variable. I looked at the source code to find out, and copied that information into the documentation.

### CODE REVIEW

File: docs/content/configuration/logging.md
Comment (Ullaakut) [inline docs/content/configuration/logging.md:13]: Personally I don't think it's relevant to mention what logger is used, and if we want to specify the accepted values, which IMO is obvious enough, I'd simplify it to:

```suggestion
Supported log levels are `trace`, `debug`, `info`, `warn`, `error`, `fatal` and `panic`.
```

But again, personally I don't think it's relevant to list that here, since it's universally the same log levels in pretty much any logger in Go, besides maybe trace and panic which might be slightly unusual but I would argue no one would likely use to run Athens unless they know what they are doing in which case they don't really need this extra line in the documentation.

**Review by Ullaakut — CHANGES_REQUESTED** [review]
Thank you @raxod502-plaid for your contribution!

Unfortunately like I explain in my comment, I would rather not merge this, but if someone thinks it's relevant, I wouldn't oppose it. If no one reaches out within a few days I'll close the PR.

File: docs/content/configuration/logging.md
Comment (nrwiersma) [reply docs/content/configuration/logging.md:13]: @matt0x6F Do you have an opinion on this?

Comment (raxod502-plaid) [reply docs/content/configuration/logging.md:13]: > unless they know what they are doing

Why would we assume that all people who deploy Athens are actively configuring low-level logging libraries for Golang in their regular work? Those seem like kind of unrelated work areas to me?

And why would we not want to clarify explicitly what standard Athens is following here, as opposed to leaving the user to guess?

Comment (matt0x6F) [reply docs/content/configuration/logging.md:13]: Athens' users have, historically, been people that work with Go but that could be changing. The change looks fine enough.

**Review by matt0x6F — APPROVED** [review]

**Pull request merged**
